app/src/components/market/common/market_data/index.tsx

```diff
@@ -89,26 +95,46 @@ export const MarketData: React.FC<Props> = props => {
       ? formatBigNumber(runningDailyVolumeByHour[Math.floor(Date.now() / (1000 * 60 * 60)) % 24], currency.decimals)
       : '0'
 
+  const totalVolume = formatBigNumber(collateralVolume, currency.decimals)
+
   return (
     <MarketDataWrapper>
       <MarketDataItem>
         <MarketDataItemTop>
-          {moment(resolutionTimestamp).format('DD.MM.YYYY - H:mm zz')} {timezoneAbbr}
+          {liquidity} {currency.symbol}
         </MarketDataItemTop>
-        <MarketDataItemBottom>Closing Date</MarketDataItemBottom>
+        <MarketDataItemBottom>Liquidity</MarketDataItemBottom>
       </MarketDataItem>
       <MarketDataItem>
         <MarketDataItemTop>
-          {resolutionTimestamp > new Date() ? moment(resolutionTimestamp).fromNow(true) : '0 days'}
+          <MarketDataItemImage src={currencyIcon && currencyIcon}></MarketDataItemImage>
```

Comment:
Looks like the currency icon isn't included in the design, so I would omit this unless @pimato has suggested otherwise

Reply:
ok

Reply:
until we did not implemented the Currency picker for an market we should probably keep it for now.